apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepConfirmation.tsx

```diff
@@ -75,11 +87,14 @@ function StepConfirmation({
           currencyName={currencyName}
         />
         {signed ? (
-          <BroadcastErrorDisclaimer
-            title={<Trans i18nKey="send.steps.confirmation.broadcastError" />}
+          <NodeError
+            error={error}
+            ticker={ticker}
+            networkName={String(mainAccount?.currency.name)}
```

Comment:
Got it I'll use the same data in llm.

I just have one last suggestion: as I wrote here https://github.com/LedgerHQ/ledger-live/pull/8081#issuecomment-2419967707 I'm passing the network and coin to `TransactionBroadcastError` to keep on re-using GenericErrorView. I made these field optionnal but maybe it would be more consistent to pass them in the here too:
```ts
error={createTransactionBroadcastError(err, {
  coin: ticker,
  network: String(mainAccount?.currency.name),
}
```
But only if you think it makes sense and it doesn't complicate things.

Reply:
Done ✅